ldaptor/test/test_attributeset.py

```diff
@@ -43,26 +42,31 @@ class TestSetOperations(unittest.TestCase):
     def testDifference(self):
         a = attributeset.LDAPAttributeSet('k', ['b', 'c', 'd'])
         b = attributeset.LDAPAttributeSet('k', ['b', 'c', 'e'])
-        self.assertEqual(a - b, sets.Set(['d']))
+        self.assertEqual(a - b, {'d'})
```

Comment:
If ldaptor still supports 2.6, `set`s cannot be defined like this.
It's probably a good time to drop 2.6 support while gaining 3.x support?

Reply:
Thanks for the review.

I don't think that 2.6 is supported... and is hard to support both 2.6 and python 3

And 2.6 is no longer supported by upstream Python for a very long time

I see that tests are executed only under 2.7 https://github.com/twisted/ldaptor/commits/master/.travis.yml

I think 2.6 was already silently dropped :)

I see that in 14.0.0 it states that 2.6 is supported but it was removed without a changelog by @psi29a here https://github.com/twisted/ldaptor/commit/b6b4ea6fc55597977dbf5a64a6c8dca44eb12fe3#diff-354f30a63fb0907d4ad57269548329e3 more than 1 year ago

Reply:
Then, awesome work!

Reply:
Twisted dropped 2.6 support, so we should follow their example.